### ISSUE TITLE
1.3-modules-beta.1: Cannot read property 'fileOptions' of undefined

### ISSUE THREAD

**Issue by nilnullzip** [meteor/meteor#5850]
Trying to update to: 1.3-modules-beta.1, I get this error:

```
4pi:boss juan$ PACKAGE_DIRS=../packages:../atomic-react/app/packages meteor update --release 1.3-modules-beta.1
boss-react: updating npm dependencies -- externalify, material-ui, react-tap-event-plugin, react-router, redux-router, redux, react-redux, redux-rx, redux-devtools, recompose, rx, lodash,
meteor-recompose, rx-react, react-select, classnames, history...
=> Errors while initializing project:         
                                              
While building package boss-bitcore:
/Users/juan/.meteor/packages/meteor-tool/.1.1.11-modules.1.ioz3sb++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/isobuild/compiler-deprecated-compile-step.js:355:43: Cannot read property
'fileOptions' of undefined (compiling bower.json)
at Object.compileStep.addJavaScript
(/Users/juan/.meteor/packages/meteor-tool/.1.1.11-modules.1.ioz3sb++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/isobuild/compiler-deprecated-compile-step.js:355:43)
at <runJavaScript-77>:203:23
at Array.forEach (native)
at Function._.each._.forEach (<runJavaScript-75>:147:11)
at <runJavaScript-77>:186:9
at Array.forEach (native)
at Function._.each._.forEach (<runJavaScript-75>:147:11)
at bowerHandler (<runJavaScript-77>:146:7)
at Package (<runJavaScript-77>:314:10)
```

**Comment by jiku** [comment]
Maybe related to #5799.

**Comment by avital** [comment]
@nilnullzip This should be fixed at `1.3-modules-beta.2`. Can you confirm?

**Comment by benjamn** [comment]
Should be fixed by https://github.com/meteor/meteor/commit/4e94b70e7643f1d82c9b037b1e390ce0823b942d.

**Comment by nilnullzip** [comment]
Works better. However now I get this one:

```
W20160112-22:31:17.556(-8)? (STDERR) 
W20160112-22:31:17.556(-8)? (STDERR) /Users/juan/.meteor/packages/meteor-tool/.1.1.11-modules.3.z5w47s++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:245
W20160112-22:31:17.556(-8)? (STDERR) 						throw(ex);
W20160112-22:31:17.556(-8)? (STDERR) 						      ^
W20160112-22:31:17.568(-8)? (STDERR) ReferenceError: module is not defined
W20160112-22:31:17.568(-8)? (STDERR)     at _URI (meteor://💻app/packages/olragon_uri-js/packages/olragon_uri-js.js:27:1)
W20160112-22:31:17.568(-8)? (STDERR)     at Package (meteor://💻app/packages/olragon_uri-js/packages/olragon_uri-js.js:35:1)
W20160112-22:31:17.568(-8)? (STDERR)     at Package (meteor://💻app/packages/olragon_uri-js/packages/olragon_uri-js.js:2018:1)
W20160112-22:31:17.568(-8)? (STDERR)     at /Users/juan/Temp/programming/hedgy/boss/boss/.meteor/local/build/programs/server/packages/olragon_uri-js.js:2035:4
W20160112-22:31:17.568(-8)? (STDERR)     at /Users/juan/Temp/programming/hedgy/boss/boss/.meteor/local/build/programs/server/packages/olragon_uri-js.js:2042:3
W20160112-22:31:17.568(-8)? (STDERR)     at /Users/juan/Temp/programming/hedgy/boss/boss/.meteor/local/build/programs/server/boot.js:242:10
W20160112-22:31:17.569(-8)? (STDERR)     at Array.forEach (native)
W20160112-22:31:17.569(-8)? (STDERR)     at Function._.each._.forEach (/Users/juan/.meteor/packages/meteor-tool/.1.1.11-modules.3.z5w47s++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160112-22:31:17.569(-8)? (STDERR)     at /Users/juan/Temp/programming/hedgy/boss/boss/.meteor/local/build/programs/server/boot.js:137:5
=> Exited with code: 8
```

**Comment by benjamn** [comment]
The `module is not defined` problem was fixed along with #5870, I believe.
